.github/linters/.yaml-lint.yml

```diff
@@ -0,0 +1,9 @@
+---
+
```

Comment:
How is this configuration file picked up by yaml-lint/super-linter? I see no mention of it in the workflow or the documentation, but I've probably just missed it.